timeseries/src/autogluon/timeseries/models/multi_window/multi_window_model.py

```diff
@@ -166,6 +174,15 @@ def _fit(
                 }
             )
 
+        # Refit the model on all windows
+        if self._refit_full:
+            model.fit(
+                train_data=train_fold,
+                val_data=val_fold,
+                time_limit=time_left_for_window,
+                **kwargs,
+            )
```

Comment:
This was my attempt at implementing the `_refit_full` logic. I hoped that by simply fitting the model on the full training data, I would get the desired result. But when I test this, the overall test error actually decreases. So this is not working.